### PR TITLE
Implemented register component

### PR DESCRIPTION
**Pull request by trowacat**
This is in response to #3 

Implemented a register component in the same style as the current sign in component. 

The signInSignUp page is named in camel case which causes the page to be unreachable from the nav as it is currently configured. Not sure if this was intentional, therefore I left it.

### CODE REVIEW

**Comment by joelwass** [comment]
If possible I'd try to refactor the existing loginForm to accept an additional field (full name) and then just have the button do something different (register, which you can leave as a TODO since it's API related), instead of copy pasting the sign in form. 

thoughts on that? I think it's just adding a "sign up" button to the login form, adding a "registering" field to state (boolean), and then toggling an extra input field (full name) based on the state of the component.

**Comment by trowacat** [comment]
@joelwass Thank you, I've made changes following your suggestion. Much better approach.

File: components/loginform.js [hunk 1]
```diff
@@ -33,8 +39,21 @@ class LoginForm extends Component {
             switchTwoClicked={() => this.updateLoginType('teacher')}
           />
         </div>
+        {/*TODO SETUP APPROPRIATE ACTION CALL FOR SIGNIN OR SIGNUP*/}
```
Comment (joelwass) [inline components/loginform.js]: linter failed, expects space after /* and before */

File: components/loginform.js [hunk 2]
```diff
@@ -33,8 +39,21 @@ class LoginForm extends Component {
...
         <form action='api/user/login' method='post'>
           <div className='panel'>
+            {this.state.register &&
```
Comment (joelwass) [inline components/loginform.js]: trailing space not allowed.

you can run `npm run lint` and `npm run fix` to auto lint

**Pull request merged**